src/model/application/mod.rs

```diff
@@ -124,7 +124,8 @@ enum_number! {
 #[cfg(feature = "unstable_discord_api")]
 #[derive(Clone, Debug, Deserialize, Serialize)]
 pub struct InstallationContextConfig {
-    pub oauth2_install_params: InstallParams,
+    #[serde(default)]
+    pub oauth2_install_params: InstallParams, // TODO: Replace with an option instead of #[serde(default)]
```

Comment:
```suggestion
    // TODO: Replace with an option instead of #[serde(default)]
    #[serde(default)]
    pub oauth2_install_params: InstallParams,
```
Formatting nit